### ISSUE TITLE
1.20.4: Hex Codes aren't Working

### ISSUE THREAD

**Issue by RocketManKian** [kangarko/ChatControl#2686]
### "/version ChatControlRed" - plugin version

10.26.3

### Are you using MySQL?

No

### Are you using BungeeCord?

No

### "/chc debug" output (strongly recommended)

[debug.zip](https://github.com/kangarko/ChatControl-Red/files/15405281/debug.zip)

### Information about the issue/bug

Not sure when this issue first arose but for some reason ChatControlRed is completely breaking Hex Codes in chat. My chat is not formatting correctly and chat is now looking like the attached image
![image](https://github.com/kangarko/ChatControl-Red/assets/47107486/8b9c94c6-cba1-4665-b1b5-1242b5eeff6a)

**Comment by kangarko** [comment]
Sorry, I cannot reproduce with channels disabled. Must be one of your other plugins causing an incompatibility.

![a](https://i.imgur.com/IF2wdmX.png)

**Comment by RocketManKian** [comment]
> Sorry, I cannot reproduce with channels disabled. Must be one of your other plugins causing an incompatibility.
> 
> ![a](https://camo.githubusercontent.com/ea14d5d4f23ca7311e3262389a3bec6bdbe519e6d6d5a94777b465ec11abc987/68747470733a2f2f692e696d6775722e636f6d2f49463277646d582e706e67)

Hey there it appears a few others are experiencing this issue on the Discord Sever. It's strange how you were unable to reproduce though, that does make me confused. But all I know is it only happens for me when ChatControl is installed.

**Comment by kangarko** [comment]
Sorry for the delay. Can you try with no other plugins installed just chatcontrol?

**Comment by kangarko** [comment]
To isolate the issue

**Comment by RocketManKian** [comment]
Hey there! So I've just given this a go with my Chat Color plugin installed as well as ChatControlRed and it's dependencies and I can confirm that I replicated your above screenshot and it seems to work with the hashtags you showed.

I actually just realised the issue and that's because I didn't realise ChatControlRed automatically converted # to colour codes whereas my Chat Colour plugin was doing the same thing so I've now made it so it just posts the Hashtags instead with no conversion and now it works a charm.

My apologies! Thank you for the help
